src/item.cpp

```diff
@@ -3549,16 +3661,21 @@ void item::final_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
 
     insert_separation_line( info );
 
+    bool any_encumb_increase = false;
     if( parts->test( iteminfo_parts::BASE_RIGIDITY ) ) {
         if( const islot_armor *t = find_armor_data() ) {
-            if( t->max_encumber != t->encumber ) {
+            any_encumb_increase = std::any_of( t->data.begin(), t->data.end(),
+            []( random_armor_data data ) {
+                return data.encumber != data.max_encumber;
+            } );
+            if( !any_encumb_increase && !contents.all_pockets_rigid() ) {
                 info.emplace_back( "BASE",
                                    _( "* This item is <info>not rigid</info>.  Its"
-                                      " volume and encumbrance increase with contents." ) );
-            } else if( !contents.all_pockets_rigid() ) {
+                                      " volume increases with contents." ) );
+            } else if( any_encumb_increase ) {
                 info.emplace_back( "BASE",
                                    _( "* This item is <info>not rigid</info>.  Its"
-                                      " volume increases with contents." ) );
+                                      " volume and encumbrance increase with contents." ) );
             }
```

Comment:
If you switch these two around it would be less code
```
if (any_encumb_increase ) {
    ...
} else if ( !contents.all_pockets_rigid() ) {
    ...
}
```